allauth/socialaccount/providers/kakao/provider.py

```diff
@@ -6,15 +6,15 @@
 
 class KakaoAccount(ProviderAccount):
     @property
-    def properties(self):
-        return self.account.extra_data.get("properties")
+    def profile(self):
+        return self.account.extra_data.get("kakao_account", {}).get("profile")
 
     def get_avatar_url(self):
-        return self.properties.get("profile_image")
+        return self.profile.get("profile_image_url")
```

Comment:
Suppose there are existing accounts with the original (legacy) data, then the changes above would crash. I think we should fallback to the original fields in case the new lookups fail?

Reply:
I fixed the code to use both profile and properties.